### ISSUE TITLE
Please add support for 3rd battery

### ISSUE THREAD

**Issue by Thebasestar** [binsentsu/home-assistant-solaredge-modbus#149]
Hi, can you please add support for 3rd battery? i own a setup with se6000h+3x se batterybank and i have no way to setup energy dashboard on HA cause of missing data from 3rd battery (calculated sensor etc )
My setup is all modbus connected (no energynet) and i can see values of all 3 batteries on SE cloud monitoring platform. (read somewhere 3rd battery datas are not available from modbus)

Many thanks for your great job

**Comment by jovana** [comment]
You can check this more up to date repo, probably it has already support for the 3rd battery: https://github.com/WillCodeForCats/solaredge-modbus-multi/discussions/296

**Comment by mpredfearn** [comment]
Nope, unfortunately this is a limitation of the (published) modbus register specification from Solaredge. There is no third battery in the specification and therefore it is impossible to support.

**Comment by regardts-hub** [comment]
> Nope, unfortunately this is a limitation of the (published) modbus register specification from Solaredge. There is no third battery in the specification and therefore it is impossible to support.

The 3rd battery has an offset of +0x0200 on top of battery 2. I.e. Battery 3 State of Energy = 0xe484

**Comment by Thebasestar** [comment]
> > Nope, unfortunately this is a limitation of the (published) modbus register specification from Solaredge. There is no third battery in the specification and therefore it is impossible to support.
> 
> The 3rd battery has an offset of +0x0200 on top of battery 2. I.e. Battery 3 State of Energy = 0xe484

Great, you made my day!!!!!
Lets hope binsetsu implement it soon.
Thanks again

**Comment by Thebasestar** [comment]
Well for everybody needings, i made this modification by myself, and it work well adding battery 3 values to homeassistant.
I'm not an expert of github, i ve just made copy paste adding strings missing for 3rd battery, but i'd  like to share to all if possible,
@binsentsu i'm here if you want.

**Comment by binsentsu** [comment]
If ready, you van make a pull request

**Comment by Thebasestar** [comment]
Done, right i hope
